mcodingbot/plugins/info.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
+
```

Comment:
Although this should probably be done at some point, it's outside the scope of this PR.